packages/venia-ui/lib/components/MagentoRoute/magentoRoute.js

```diff
@@ -34,14 +35,14 @@ const MagentoRoute = () => {
     } else if (routeError === NOT_FOUND) {
         return (
             <ErrorView>
-                <h1>{MESSAGES.get(routeError)}</h1>
+                <h1>{formatMessage({ id: MESSAGES.get(routeError) })}</h1>
             </ErrorView>
         );
     }
 
     return (
         <ErrorView>
-            <h1>{MESSAGES.get(INTERNAL_ERROR)}</h1>
+            <h1>{formatMessage({ id: MESSAGES.get(INTERNAL_ERROR) })}</h1>
```

Comment:
Same idea here. We can use the map strings as a default message but should also make sure they are in the json.

Reply:
Done